llms/openai/openaillm.go

```diff
@@ -10,7 +10,7 @@ import (
 	"github.com/tmc/langchaingo/llms/openai/internal/openaiclient"
 )
 
-const tokenEnvVarName = "OPENAI_API_KEY"
+var tokenEnvVarName = os.Getenv("OPENAI_API_KEY")
```

Comment:
this seems to change the semantics here -> EnvVarName vs EnvVar value?

Reply:
I would have named it something else I guess, but I was just trying to make it pass a gosec check.

Reply:
oh i see what you're saying, let me double check that

Reply:
fixed